pkg/plugin/v3/scaffolds/internal/templates/config/manager/manager_config.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manager
+
+import (
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/v2/pkg/model/file"
+)
+
+var _ file.Template = &ControllerManagerConfig{}
+
+// ControllerManagerConfig scaffolds the ComponentConfig file in manager folder.
+type ControllerManagerConfig struct {
```

Comment:
I guess we can have it like this (Linter doesn't allow to call it `ManagerConfig` as it is inside the `manager` package) until we clean the whole templates.

Reply:
Sorry about that, I pushed before renaming the file to match.